docs/src/APIs/Numerics/DGMethods/Courant.md

```diff
@@ -0,0 +1,13 @@
+# Discontinuous Galerkin Methods
+
+```@meta
+CurrentModule = ClimateMachine
+```
+
+## Functions
+
+```@docs
+Courant.advective_courant
```

Comment:
```suggestion
Courant
Courant.advective_courant
```
The complaint is about the docstring for the module, which is added with simply `ClimateMachine.Courant`.